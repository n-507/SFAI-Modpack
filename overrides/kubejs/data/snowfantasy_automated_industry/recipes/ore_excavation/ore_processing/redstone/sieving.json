{
  "type": "createsifter:sifting",
  "ingredients": [
    {
      "item": "kubejs:redstone_batch"
    },
    {
      "item": "createsifter:brass_mesh"
    }
  ],
  "processingTime": 300,
  "results": [
    {
      "chance": 0.9,
      "item": "minecraft:redstone"
    },
    {
      "chance": 0.8,
      "item": "minecraft:redstone"
    },
    {
      "chance": 0.7,
      "item": "minecraft:redstone"
    },
    {
      "chance": 0.6,
      "item": "minecraft:redstone"
    },
    {
      "chance": 0.5,
      "item": "minecraft:redstone"
    },
    {
      "chance": 0.5,
      "item": "ae2:certus_quartz_crystal"
    },
    {
      "chance": 0.3,
      "item": "powah:uraninite_raw"
    }
  ]
}