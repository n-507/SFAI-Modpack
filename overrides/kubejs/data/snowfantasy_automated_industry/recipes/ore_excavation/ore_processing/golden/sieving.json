{
  "type": "createsifter:sifting",
  "ingredients": [
    {
      "item": "kubejs:golden_batch"
    },
    {
      "item": "createsifter:brass_mesh"
    }
  ],
  "processingTime": 300,
  "results": [
    {
      "chance": 0.3,
      "item": "minecraft:raw_gold"
    },
    {
      "chance": 0.9,
      "item": "thermal:raw_silver"
    },
    {
      "chance": 0.2,
      "item": "mekanism:raw_uranium"
    },
    {
      "chance": 0.6,
      "item": "mekanism:raw_osmium"
    },
    {
      "chance": 0.5,
      "item": "minecraft:gold_nugget"
    }
  ]
}